src/locales/ar.po

```diff
@@ -1221,10 +1221,10 @@ msgid "Website exclusion is currently set to <strong>%1.</strong>"
 msgstr "استبعاد الموقع مضبوط حالياً على <strong>%1.</strong>"
 
 msgid "Win payout if %3 after %plural(%5,%d tick,%d ticks) is higher than or equal to %6."
-msgstr "أربح عائد إذا %3 بعد % الجمع(%5, علامة دي%, علامات دي%) أعلى من أو تساوي ل %6."
+msgstr "أربح عائد إذا %3 بعد %plural(%5,%d علامة,%d علامات) أعلى من أو تساوي ل %6."
```

Comment:
Yes, I think the plural forms will need fixing throughout the file.

The RTL switches seem to be confusing the display (and me), that extra ) is towards the end of the line. The string shows up as "علامات)" in the github email and vim, but github web UI and gedit show the text in RTL mode, so the ) is reversed and at the end of the line. Mismatched  LTR/RTL pairs maybe? If so, perhaps this can be added to the Locale::Maketext::Test checks.